### ISSUE TITLE
I'd be interested in taking over if you'd like!

### ISSUE THREAD

**Issue by iCodeForBananas** [twood02/atometer#6]
@twood02 Idk how you'd feel about this but I'd love to maintain this project.

**Comment by twood02** [comment]
Sure, I never got too far with it. Let me know how you'd like to do so--I can give you access to the repo, or we may be able to deregister this plugin and add a new one you host somewhere. 

I haven't tried to run the code in a couple years, so I assume it is very broken

**Comment by iCodeForBananas** [comment]
@twood02 It still works!

I forked it here and just haven't added the credit yet.
https://github.com/michaeljcalkins/keys-typed-tracker
https://atom.io/packages/keys-typed-tracker

**Comment by twood02** [comment]
Cool! Have fun with it.  Before I moved on to other things I noticed that the data would sometimes get lost if Atom was started in different folders.  My plan had been to move the data into a proper database (eg sqlite) so I could support things like graphs of behavior over time, but obviously I never got around to that.

If you want to take over the "atometer" name let me know and I can try to disable my current version.
